contrib/cr3bp_DhruvJ/cr3bp_char_quant.py

```diff
@@ -0,0 +1,156 @@
+"""
+Created on 10 Nov 2021 22:43:38 2022
+
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+        dhruvj9922@gmail.com
+
+Objective: This file contains functions to compute various characteristic quantities
+           for a Cirular Restircted Three Body Problem (CR3BP) model setup
+           
+           Features: 
+               1. Compute CR3BP 'mu', characterisitic mass of P1-P2 primary bodies
```

Comment:
If I understand correctly, **characteristic mass** is M* = M1 + M2
and the right term for mu should be **mass ratio** or **CR3BP constant**. Could you double-check this?

Reply:
yup, you are right, my bad.